2024-07-20 05:11:30.520 WARNING fixture:conftest.py:60: Process started
2024-07-20 05:11:30.520 INFO fixture:conftest.py:61: Process ongoing..
2024-07-20 05:11:30.523 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.address`.
2024-07-20 05:11:30.524 DEBUG faker.factory:factory.py:95: Provider `faker.providers.address` has been localized to `en_US`.
2024-07-20 05:11:30.525 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.automotive`.
2024-07-20 05:11:30.526 DEBUG faker.factory:factory.py:95: Provider `faker.providers.automotive` has been localized to `en_US`.
2024-07-20 05:11:30.526 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.bank`.
2024-07-20 05:11:30.527 DEBUG faker.factory:factory.py:86: Specified locale `en_US` is not available for provider `faker.providers.bank`. Locale reset to `en_GB` for this provider.
2024-07-20 05:11:30.527 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.barcode`.
2024-07-20 05:11:30.527 DEBUG faker.factory:factory.py:95: Provider `faker.providers.barcode` has been localized to `en_US`.
2024-07-20 05:11:30.527 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.color`.
2024-07-20 05:11:30.529 DEBUG faker.factory:factory.py:95: Provider `faker.providers.color` has been localized to `en_US`.
2024-07-20 05:11:30.529 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.company`.
2024-07-20 05:11:30.530 DEBUG faker.factory:factory.py:95: Provider `faker.providers.company` has been localized to `en_US`.
2024-07-20 05:11:30.530 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.credit_card`.
2024-07-20 05:11:30.530 DEBUG faker.factory:factory.py:95: Provider `faker.providers.credit_card` has been localized to `en_US`.
2024-07-20 05:11:30.531 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.currency`.
2024-07-20 05:11:30.531 DEBUG faker.factory:factory.py:95: Provider `faker.providers.currency` has been localized to `en_US`.
2024-07-20 05:11:30.531 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.date_time`.
2024-07-20 05:11:30.532 DEBUG faker.factory:factory.py:95: Provider `faker.providers.date_time` has been localized to `en_US`.
2024-07-20 05:11:30.532 DEBUG faker.factory:factory.py:106: Provider `faker.providers.emoji` does not feature localization. Specified locale `en_US` is not utilized for this provider.
2024-07-20 05:11:30.532 DEBUG faker.factory:factory.py:106: Provider `faker.providers.file` does not feature localization. Specified locale `en_US` is not utilized for this provider.
2024-07-20 05:11:30.532 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.geo`.
2024-07-20 05:11:30.533 DEBUG faker.factory:factory.py:95: Provider `faker.providers.geo` has been localized to `en_US`.
2024-07-20 05:11:30.533 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.internet`.
2024-07-20 05:11:30.534 DEBUG faker.factory:factory.py:95: Provider `faker.providers.internet` has been localized to `en_US`.
2024-07-20 05:11:30.534 DEBUG faker.factory:factory.py:106: Provider `faker.providers.isbn` does not feature localization. Specified locale `en_US` is not utilized for this provider.
2024-07-20 05:11:30.534 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.job`.
2024-07-20 05:11:30.535 DEBUG faker.factory:factory.py:95: Provider `faker.providers.job` has been localized to `en_US`.
2024-07-20 05:11:30.535 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.lorem`.
2024-07-20 05:11:30.535 DEBUG faker.factory:factory.py:95: Provider `faker.providers.lorem` has been localized to `en_US`.
2024-07-20 05:11:30.535 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.misc`.
2024-07-20 05:11:30.536 DEBUG faker.factory:factory.py:95: Provider `faker.providers.misc` has been localized to `en_US`.
2024-07-20 05:11:30.536 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.passport`.
2024-07-20 05:11:30.536 DEBUG faker.factory:factory.py:95: Provider `faker.providers.passport` has been localized to `en_US`.
2024-07-20 05:11:30.536 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.person`.
2024-07-20 05:11:30.538 DEBUG faker.factory:factory.py:95: Provider `faker.providers.person` has been localized to `en_US`.
2024-07-20 05:11:30.539 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.phone_number`.
2024-07-20 05:11:30.541 DEBUG faker.factory:factory.py:95: Provider `faker.providers.phone_number` has been localized to `en_US`.
2024-07-20 05:11:30.541 DEBUG faker.factory:factory.py:106: Provider `faker.providers.profile` does not feature localization. Specified locale `en_US` is not utilized for this provider.
2024-07-20 05:11:30.541 DEBUG faker.factory:factory.py:106: Provider `faker.providers.python` does not feature localization. Specified locale `en_US` is not utilized for this provider.
2024-07-20 05:11:30.541 DEBUG faker.factory:factory.py:106: Provider `faker.providers.sbn` does not feature localization. Specified locale `en_US` is not utilized for this provider.
2024-07-20 05:11:30.541 DEBUG faker.factory:factory.py:76: Looking for locale `en_US` in provider `faker.providers.ssn`.
2024-07-20 05:11:30.543 DEBUG faker.factory:factory.py:95: Provider `faker.providers.ssn` has been localized to `en_US`.
2024-07-20 05:11:30.543 DEBUG faker.factory:factory.py:106: Provider `faker.providers.user_agent` does not feature localization. Specified locale `en_US` is not utilized for this provider.
2024-07-20 05:11:30.544 DEBUG urllib3.connectionpool:connectionpool.py:1048: Starting new HTTPS connection (1): dev.agentum.pro:443
2024-07-20 05:11:31.177 DEBUG urllib3.connectionpool:connectionpool.py:546: https://dev.agentum.pro:443 "POST /backend/api/deal/create-with-client HTTP/1.1" 200 None
2024-07-20 05:11:31.177 CRITICAL root:helpers_api.py:27: ___response code- 200
2024-07-20 05:11:31.177 CRITICAL root:helpers_api.py:28: ___response body- {'deal_id': 5304}
2024-07-20 05:11:31.179 INFO fixture:conftest.py:63: Process ended